master/buildbot/process/properties.py

```diff
@@ -460,6 +460,16 @@ def getRenderingFor(self, build):
 _thePropertyDict = _PropertyDict()
 
 
+@implementer(IRenderable)
```

Comment:
Need a test! Unit test, and integration test (template example is in https://github.com/buildbot/buildbot/blob/master/master/buildbot/test/integration/test_integration_template.py)

Reply:
Yeah once the actual code stabilizes.

Reply:
Having some running tests will make details like unicode more obvious, so it is good to work on the tests at the same time (aka TDD)